<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>GeometryFactory</author>
 <class>Periodic_4_hyperbolic_Delaunay_triangulation_2</class>
 <widget class="QMainWindow" name="Periodic_4_hyperbolic_Delaunay_triangulation_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Periodic hyperbolic triangulation demo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/cgal/logos/cgal_icon</normaloff>:/cgal/logos/cgal_icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>File Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClear"/>
   <addaction name="actionLoadPoints"/>
   <addaction name="actionSavePoints"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Visualization Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlayDemo"/>
   <addaction name="actionInsertPoint"/>
   <addaction name="actionCircumcenter"/>
   <addaction name="actionShowConflictZone"/>
   <addaction name="separator"/>
   <addaction name="actionShowTriangulation"/>
   <addaction name="actionShowVoronoi"/>
   <addaction name="actionShowOctagon"/>
   <addaction name="separator"/>
   <addaction name="actionRecenter"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="actionLoadPoints"/>
    <addaction name="actionSavePoints"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionInsertRandomPoints"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionPlayDemo"/>
    <addaction name="actionInsertPoint"/>
    <addaction name="actionCircumcenter"/>
    <addaction name="actionShowConflictZone"/>
    <addaction name="separator"/>
    <addaction name="actionShowTriangulation"/>
    <addaction name="actionShowVoronoi"/>
    <addaction name="actionShowOctagon"/>
    <addaction name="separator"/>
    <addaction name="actionRecenter"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutCGAL">
   <property name="text">
    <string>About &amp;CGAL</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInsertRandomPoints">
   <property name="text">
    <string>&amp;Insert random points</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionPlayDemo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Main_resources.qrc">
     <normaloff>:/icon/icons/play.png</normaloff>:/icon/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Play Demo</string>
   </property>
   <property name="toolTip">
    <string>Play Demo</string>
   </property>
   <property name="statusTip">
    <string>Left: Play demo</string>
   </property>
  </action>
  <action name="actionInsertPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Main_resources.qrc">
     <normaloff>:/icon/icons/inputPoint.png</normaloff>:/icon/icons/inputPoint.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Insert Point</string>
   </property>
   <property name="toolTip">
    <string>Insert Point</string>
   </property>
   <property name="statusTip">
    <string>Left: Insert vtx</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="Main_resources.qrc">
     <normaloff>:/icon/icons/fileNew.png</normaloff>:/icon/icons/fileNew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionShowOctagon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Main_resources.qrc">
     <normaloff>:/icon/icons/octagon.png</normaloff>:/icon/icons/octagon.png</iconset>
   </property>
   <property name="text">
    <string>Show/hide &amp;Octagon</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionShowConflictZone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Main_resources.qrc">
     <normaloff>:/icon/icons/conflict_zone.png</normaloff>:/icon/icons/conflict_zone.png</iconset>
   </property>
   <property name="text">
    <string>Show Con&amp;flict Zone</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionShowVoronoi">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Main_resources.qrc">
     <normaloff>:/icon/icons/Voronoi_diagram_2.png</normaloff>:/icon/icons/Voronoi_diagram_2.png</iconset>
   </property>
   <property name="text">
    <string>Show &amp;Voronoi Diagram</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionShowTriangulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Main_resources.qrc">
     <normaloff>:/icon/icons/triangulation.png</normaloff>:/icon/icons/triangulation.png</iconset>
   </property>
   <property name="text">
    <string>Show &amp;Delaunay Triangulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionLoadPoints">
   <property name="icon">
    <iconset resource="Main_resources.qrc">
     <normaloff>:/icon/icons/fileOpen.png</normaloff>:/icon/icons/fileOpen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Points...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSavePoints">
   <property name="icon">
    <iconset resource="Main_resources.qrc">
     <normaloff>:/icon/icons/fileSave.png</normaloff>:/icon/icons/fileSave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Points...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCircumcenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Main_resources.qrc">
     <normaloff>:/icon/icons/circumcenter.png</normaloff>:/icon/icons/circumcenter.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Circumcenter</string>
   </property>
   <property name="toolTip">
    <string>Draw circumcenter</string>
   </property>
  </action>
  <action name="actionRecenter">
   <property name="icon">
    <iconset resource="Main_resources.qrc">
     <normaloff>:/icon/icons/fit-page-32.png</normaloff>:/icon/icons/fit-page-32.png</iconset>
   </property>
   <property name="text">
    <string>Re&amp;center the viewport</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Main_resources.qrc"/>
 </resources>
 <connections/>
</ui>
